stdlib/securerandom.rb

```diff
@@ -1,4 +1,35 @@
 module SecureRandom
+  def self.hex(count = nil)
+    count ||= 16
+    count = count.to_int unless `typeof count === "number"`
+    raise ArgumentError, 'count of hex numbers must be positive' if count < 0
+    %x{
+      count = Math.floor(count);
+      var repeat = Math.floor(count / 6),
+          remain = count % 6,
+          remain_total = remain * 2,
+          string = '',
+          temp;
+      for (var i = 0; i < repeat; i++) {
+        temp = Math.floor(Math.random()*parseInt('ff'.repeat(6), 16)).toString(16);
```

Comment:
Can you replace `parseInt('ff'.repeat(6), 16)` with the static result?
The original code can be added as a comment just above with a brief explanation on how it's been obtained.

Reply:
Done.